pkg/diff/diff.go

```diff
@@ -97,6 +98,42 @@ func DiffUnstructureds(x, y unstructured.Unstructured, key string, opts ...Optio
 	return d.result, nil
 }
 
+// DiffStructs calulates the diff between two struct objects,
+// which are not k8s manifests.
```

Comment:
I can imagine this related to this line in your PR description.

> Including Diff() func, which compares manifests(i.e. serviceDef&taskDef in ECS) for non-k8s resources like Lambda

But it is still hard to imagine when to use `DiffUnstructureds` and `DiffStructs`. Maybe we need a better name, or we need to add comments that describe these two functions better.

Reply:
Maybe we also should rename this to `DiffStructureds` either ways 🤔